frontend/lib/norent/letter-builder/rent-periods.tsx

```diff
@@ -26,9 +26,18 @@ function getCurrentRentNonpaymentPeriods(s: AllSessionInfo): string[] {
 function getRentNonpaymentChoices(
   periods: AllSessionInfo["norentAvailableRentPeriods"]
 ): DjangoChoices {
+  // LA residents' protections extend beyond those in other cities.
+  function addCaveatForLA(paymentDate: GraphQLDate) {
+    let caveat = "";
+    if (new Date(paymentDate) >= new Date("01 October 2021")) {
```

Comment:
Would it be helpful to save `new Date("01 October 2021")` as a constant outside of the function definition, maybe something called `END_DATE_OF_NON_LA_PROTECTIONS` or something like that, just in case it changes in the future?

Reply:
Great idea